python/lsst/ip/isr/isrTask.py

```diff
@@ -747,13 +988,19 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None, crosstalkSou
         atmosphereTransmission : `lsst.afw.image.TransmissionCurve`
             A ``TransmissionCurve`` that represents the throughput of the
             atmosphere, assumed to be spatially constant.
+        detectorNum : `int`, optional
+            The integer number for the detector to process.
+        isGen3 : bool, optional
+            Flag this call to run() as using the Gen3 butler environment.
 
         Returns
         -------
         result : `lsst.pipe.base.Struct`
             Result struct with component:
             - ``exposure`` : `afw.image.Exposure`
                 The fully ISR corrected exposure.
+            - ``outputExposure`` : `afw.image.Exposure`
+                An alias for `exposure`
```

Comment:
Why add the alias?  Is it part of the sanitization of terminology that is coming with Gen3?

Reply:
I aliased it to keep the gen2/gen3 elements of the struct independent.  Hopefully this keeps testing, debugging, and gen2 deprecation simpler.